tools/cldr-code/src/main/java/org/unicode/cldr/json/LdmlConvertRules.java

```diff
@@ -154,6 +154,7 @@
 
         // in common/bcp47/*.xml
         "keyword:key:alias",
+        "keyword:key:name",
         "key:type:alias",
         "key:type:name",
```

Comment:
may not give optimal results… let's see.

Reply:
What exactly do you mean by this? 

I added this based on our conversation in Slack where you said, 

> "Probably need to add `keyword:key:name` to one of the lists in `LdmlConvertRules.java`. Take a spin through that file and see if anything jumps out at you."

But I am not familiar enough with the entirety of this code base to know what an alternative is off the top of my head, or why this may be sub optimal, for that matter.

Reply:
@nordzilla for the record: it's because the output JSON may not be the easiest to consume.